rules/integrations/cloud_defend/discovery_suspicious_network_tool_launched_inside_a_container.toml

```diff
@@ -0,0 +1,81 @@
+[metadata]
+creation_date = "2023/04/26"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/04/26"
+
+[rule]
+author = ["Elastic"]
+description = "This rule detects commonly abused network utilities running inside a container. Network utilities like nc, nmap, dig, tcpdump, ngrep, telnet, mitmproxy, zmap can be used for malicious purposes such as network reconnaissance, monitoring or exploitation, and should be monitored closely within a container."
+from = "now-6m"
+index = [ "logs-cloud_defend*" ]
+interval = "5m"
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious Network Tool Launched Inside A Container"
+
+tags = [
+  "Elastic",
+  "Host",
+  "Linux",
+  "Threat Detection",
+  "Discovery",
+  "Command and Control",
+  "Reconnaissance",
+  "Container"
+]
+risk_score = 21
+rule_id = "1a289854-5b78-49fe-9440-8a8096b1ab50"
+severity = "low"
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = """
+process where process.entry_leader.entry_meta.type== "container" and event.type== "start" and 
+(
+(process.name: ("nc", "ncat", "nmap", "dig", "nslookup", "tcpdump", "tshark", "ngrep", "telnet", "mitmproxy", "socat", "zmap")) or 
```

Comment:
Thoughts on adding `masscan` and `zgrab`? These seem to be common open-sourced tools that adversaries have used against Docker APIs, K8s clusters and kubelets.